cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -194,8 +195,11 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
             throw ProgramResult(1)
         }
 
-        if (scanResults.hasIssues) {
-            println("The scan result contains issues.")
+        val hasIssues = scanResults.scanResults.any { (_, results) ->
```

Comment:
The name `hasIssues` is a bit misleading (as it's not about issues in general, and we already have `ScanRecord.hasIssues`), so how about `hasSevereIssues`?